canonical/ubuntu.com
# WD-11393 - remove sales numbers

## Done

Remove sales numbers

## QA

- [demo link](https://ubuntu-com-13936.demos.haus/)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the sales numbers are removed in those places:
https://ubuntu-com-13936.demos.haus/contact-us
https://ubuntu-com-13936.demos.haus/download/server/s390x
https://ubuntu-com-13936.demos.haus/hpc/contact-us
https://ubuntu-com-13936.demos.haus/pricing/pro
https://ubuntu-com-13936.demos.haus/pricing/desktop
https://ubuntu-com-13936.demos.haus/pricing/devices
https://ubuntu-com-13936.demos.haus/openstack/contact-us

## Issue / Card
[WD-11393](https://warthogs.atlassian.net/browse/WD-11393)

[WD-11393]: https://warthogs.atlassian.net/browse/WD-11393?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
